src/ifcconvert/IfcConvert.cpp

```diff
@@ -482,16 +517,25 @@ int main(int argc, char** argv) {
 
 	int old_progress = -1;
 
-	if (center_model) {
-		double* offset = serializer->settings().offset;
-		gp_XYZ center = (context_iterator.bounds_min() + context_iterator.bounds_max()) * 0.5;
-		offset[0] = -center.X();
-		offset[1] = -center.Y();
-		offset[2] = -center.Z();
-		std::stringstream msg;
-		msg << "Using model offset (" << offset[0] << "," << offset[1] << "," << offset[2] << ")";
-		Logger::Message(Logger::LOG_NOTICE, msg.str());
-	}
+    if (center_model || model_offset) {
+        double* offset = serializer->settings().offset;
+        if (center_model) {
+            gp_XYZ center = (context_iterator.bounds_min() + context_iterator.bounds_max()) * 0.5;
+            offset[0] = -center.X();
+            offset[1] = -center.Y();
+            offset[2] = -center.Z();
+        } else {
+            if (sscanf(offset_str.c_str(), "%lf;%lf;%lf", &offset[0], &offset[1], &offset[2]) != 3) {
```

Comment:
Wonder if we need to do something with `setlocale()` here? I'm somewhat surprised that I need to use a dot as a decimal separator even though my OS is set to believe in the mainland European convention of using commas. It's a minor issue and I'm sure users can figure this out for themselves.